src/components/Layout.re

```diff
@@ -1,4 +1,112 @@
-[@bs.module "./layout"] [@react.component]
-external make: (~children: React.element) => React.element = "default";
+[@bs.module "gatsby"] external useStaticQuery: string => 'a = "useStaticQuery";
+
+%bs.raw
+{|
+    import  {graphql}  from "gatsby"
+    import 'normalize.css'
+    import './layout.css'
+    import './fonts.css'
+    import './text.css'
+|};
+
+type siteMetadataType = {title: string};
+type siteType = {siteMetadata: siteMetadataType};
+type queryResponseType = {site: siteType};
+
+[@react.component]
+let make = (~children: ReasonReact.reactElement) => {
+  let data: queryResponseType =
+    useStaticQuery(
+      [%bs.raw
+        {|
+                graphql`
+                    query SiteTitleQuery {
+                      site {
+                        siteMetadata {
+                          title
+                        }
+                      }
+                    }
+                  `
+                |}
+      ],
+    );
+
+  let meta = [%bs.raw
```

Comment:
Would consider avoiding to use `bs.raw` on here. Since BuckleScript allows to map perfectly with the JS expected format.

``
`
type metaItem = { name: string, content: string };
type meta = array(metaItem);

let meta = [| { name: "description", content: "A playful avatar generator for the modern age" } |]
```

I would double-check what's the resultant JS, but BuckleScript v7.x should transform Records to JS objects.